Resources/Prototypes/Entities/Objects/Misc/pet_carrier.yml

```diff
@@ -36,7 +36,9 @@
   - type: EntityStorage
     capacity: 1
     airtight: false
-    itemCanStoreMobs: true
+    whitelist:
+      tags:
+        - VimPilot # If it can fit in a Vim it can fit in a pet carrier.
```

Comment:
Worth it to rename the `VimPilot` tag since it's used more as a "mob can be put into things" tag now?

Reply:
That would be a polluting impact of this PR, making this harder to revert in the future if needed.